OpenTranslator/OpenTranslator.csproj

```diff
@@ -300,6 +300,9 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>StringTranslation.edmx</DependentUpon>
     </Compile>
+    <Compile Include="Data\sysdiagram.cs">
```

Comment:
just for future pulls: don't commit or pull *.csproj unless you have a new classes/views/assets/packages added. This is just to avoid conflicts when merging